README.md

```diff
@@ -11,26 +11,14 @@ aims to be significantly more lightweight in its concepts.
 Also, Effekt comes with no warranty and there are (probably) many bugs -- If this does not discourage you, feel free to
 play with it and give us your feedback :)
 
-## Installation
-
-You need to have Java (>= 11) and Node (>= 16) and npm installed.
-
-The recommended way of installing Effekt is by running:
+## Examples
 
-```
-npm install -g https://github.com/effekt-lang/effekt/releases/latest/download/effekt.tgz
-```
+If you are looking for examples of the Effekt language, we invite you to look at the [casestudies](examples/casestudies/) and the [language tour](https://effekt-lang.org/docs/tour).
```

Comment:
The partial path here can sometimes be broken -- did you want people to take a look at the case studies online or in the repo? If in the repo, then
```suggestion
If you are looking for examples of the Effekt language, we invite you to look at the [casestudies](https://github.com/effekt-lang/effekt/tree/master/examples/casestudies) and the [language tour](https://effekt-lang.org/docs/tour).
```